wordpress-mobile/WordPress-Android
# Issue/13326 scan screen UI iteration 1

Parent #13326

This PR makes small UI improvements to the Scan screen for scan state items:
- Separates primary and secondary buttons
- Add icon margin, size dynamically (for different margins, sizes on scan and threat details screen)
- Adds vertical and horizontal margins to the scan screen items

To test:

That scan states UI is rendered properly:

|scan ok|scanning|threats found|fixing|
|---|---|---|---|
|![scan_okay](https://user-images.githubusercontent.com/1405144/105676249-44018d80-5f10-11eb-9dd0-8fe23560061a.jpg)|![scanning](https://user-images.githubusercontent.com/1405144/105676274-4cf25f00-5f10-11eb-9faf-e749dc4dbf12.jpg)|![threats_found](https://user-images.githubusercontent.com/1405144/105676297-54b20380-5f10-11eb-9ac3-d79579a8571f.jpg)|![fixing](https://user-images.githubusercontent.com/1405144/105676323-5aa7e480-5f10-11eb-9547-f60e15d2c8c4.jpg)

Notes:

1. Review by commits might be easier
2. Yet to add CTA for "here to help" - added to backlog

Merge Instructions

- Make sure PR #13872 is merged to develop
- Remove the "Not Ready for Merge label"
- Merge this PR

PR submission checklist:
